app/retail/templates/emails/grant_txn_failed.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n humanize %}
+
+{% block content %}
+
+<h1>Grants transaction failed, try again?</h1>
+
+<div class="grant-txn-msg">
+  <p>
+    Your contribution to <b>{{ grant_title }}</b> failed due to a technical issue.
+  </p>
+</div>
+
+<div class="grant-txn-id">
+  <p>
+    Transaction ID: <br>
+    <a href="{{ tx_url }}">{{ tx_id }}</a>
+  </p>
+</div>
+
+<div style="margin-bottom: 3em; margin-top: 3em;">
+  <a class="button" href="#">{% trans "Resubmit Transaction" %}</a> <!-- TODO: Replace correct href to add the failed grant to cart -->
```

Comment:
This needs to be fixed , the least we can do is link them to the grant so that they can add it themselves

Reply:
@owocki any suggestions? Maybe use the bulk-add link?

Reply:
yeah this needs to be updated before we deploy I think - @sebastiantf I think the bulk add link would be perfect

Reply:
@danlipert @thelostone-mc Done in https://github.com/gitcoinco/web/pull/6791/commits/902f5486a51fe028c83439595fa614a38c036094